export/auth_export/accounts.json

```diff
@@ -0,0 +1,47 @@
+{
+  "kind": "identitytoolkit#DownloadAccountResponse",
+  "users": [
+    {
+      "localId": "R7dWvQjkuhsCJgplJ3WtnUDsD9kR",
+      "displayName": "Experimenter",
+      "email": "experimenter@google.com",
+      "emailVerified": true,
+      "createdAt": "1714400539266",
+      "lastLoginAt": "1714400583451",
+      "providerUserInfo": [
+        {
+          "providerId": "google.com",
+          "rawId": "1651642867512372825802162318175962691422",
+          "federatedId": "1651642867512372825802162318175962691422",
+          "displayName": "Experimenter",
+          "email": "experimenter@google.com"
+        }
+      ],
+      "lastRefreshAt": "2024-04-29T14:23:03.451Z",
+      "salt": "fakeSalttpRdKDA4NS5sxXb2GeoT",
```

Comment:
Can these fields be skipped?

Reply:
(sald and passwordHash), I'd generally rather avoid such info in a repo, even if it's fake. Or is this needed to login to the emulator?

Reply:
fixed in f5a7d3eb062455978049c0c7aee1e253cddc5cf7